### ISSUE TITLE
Experience (EXP) / Stats proposal

### ISSUE THREAD

**Issue by h4kbas** [EmpatiLab/SoulBender#3]
## What's proposed?

SoulBender should have an advanced experience system. Basically, there has to be a few skills like in the real world. If you do something for a while, it should be better. In the beginning, they should be randomly distributed like in the real world. Some can be better at fighting but some can be better at jumping etc. 

For example, you always run so your running will be better in time even if when you are born your running sucks. Also, if you quit running for a long time, your skill will be less efficient just like in the real world. 

 
## Why it's proposed?

In order to have a expanding and gamer oriented game, we need this type of features, which gamers can shape the game according to their needs. 

## Context

In order to implement this feature, I think all the basic skills that game gives us will be multiplied with a float number and that float number should be mutated according to the activity frequency.

Also, these skills can be expanded according to the script system. We created a new item with a script / spell so we may need some specific experience point to use it.

For example, I created an item called Bow and I want a new skill called Archery to make people use it. If they do not have the minimum requirement of that skill, they cannot use it. They can either learn the basic skill from a book or from a person who has that skill already 

## Visual Attachments

[if available, include your visuals with ### heading and add a single line explanation what the visual represents]

----

## Resources

[please provide us resources to make us understand the topic deeply at least 1 needed]

1. [Example Resource Name](https://resource.addres/page)

**Comment by kaaninel** [comment]
+1

I believe we should hide players' actual stats from them. This can improve gameplay since players actually need to explore their own character.

**Comment by h4kbas** [comment]
Yeah that's quite natural, we can hide the stats so that they do not focus on the numbers but is the idea of having dynamic multiple stats applicable?
